README.md

```diff
@@ -12,27 +12,20 @@ setup instructions check the [github](https://github.com/faiface/pixel).
 
 ## Building and running
 
-First you need to install dependencies:
+First you need to install dependencies, build the project, and run with:
```

Comment:
Too many commas.

Reply:
It's correct in English language

Reply:
It is known as [Oxford comma](http://www.grammarbook.com/punctuation/commas.asp)